### ISSUE TITLE
Validate the correctness of the training (classification)

### ISSUE THREAD

**Issue by beistvan** [beistvan/leaffliction#6]

**Comment by beistvan** [comment]
There is fair enough result with with using 15 epochs and with the batch size of 64.

`Epoch 15/15 164/164 ━━━━━━━━━━━━━━━━━━━━ 186s 1s/step - accuracy: 0.9698 - loss: 0.0901 - val_accuracy: 0.9726 - val_loss: 0.0895 41/41 ━━━━━━━━━━━━━━━━━━━━ 9s 229ms/step - accuracy: 0.9693 - loss: 0.0894 `
